safeeyes/config/lang/pt.json

```diff
@@ -25,33 +25,33 @@
     },
     "ui_controls": {
         "about": "Sobre",
-        "allow_postpone": "Allow postponing the breaks",
+        "allow_postpone": "Permite o adiamento da pausa",
         "audible_alert": "Alerta sonoro no fim da pausa",
         "cancel": "Cancelar",
-        "close": "Close",
+        "close": "Fechar",
         "disable": "Desativar Safe Eyes",
         "enable": "Habilitar Safe Eyes",
-        "enable_screen_lock": "Lock the screen after every long break",
+        "enable_screen_lock": "Bloqueie a tela após cada pausa longa",
         "for_x_hour": "Por {} Hora",
         "for_x_hours": "Por {} Horas",
         "for_x_minutes": "Por {} Minutos",
         "idle_time": "Tempo mínimo de inatividade para pausar (em minutos)",
         "interval_between_two_breaks": "Intervalo entre duas pausas",
         "language": "Idioma",
-        "license": "License",
+        "license": "Licença",
         "long_break_duration": "Duração de uma pausa longa (em segundos)",
         "no_of_short_breaks_between_two_long_breaks": "Número de pausas curtas entre duas pausas longas",
         "postpone": "Postpone",
```

Comment:
Hi,

Thank you for the translation. Could you please add one more commit translating "Postpone"?

Thanks